### DISCUSSION TITLE
newbie international crowdsourcing questions

### DISCUSSION THREAD

**Discussion by abubelinha** [Symbiota/community-board#424] [General]
Hello.  I am new to symbiota and have some questions about its global scope and usage:

1. In [Symbiota portals page](https://symbiota.org/symbiota-portals/), when it says a portal is "GLOBAL", does it mean worldwide scope for occurrence records, for participating institutions or for both?   
 For example, [globaltcn.utk.edu](https://globaltcn.utk.edu) page says "is a collaboration of 25 universities, museums, and gardens **located across the _United States_ and funded by the National Science Foundation (NSF)**. This project will integrate information about bryophytes and lichens, with each other and with their commensal organisms, **_on a worldwide scale_**".  
  I have not clear if it makes any sense to you that an institution from outside North America joins that network to share records and participate in crowdsourcing. 
  It makes sense to me, but the NSF comment makes me doubt.
2. Regarding the [crowdsourcing guide](https://drive.google.com/file/d/1J5ZiqjrirlU87V-waCqd33dNF_dM9WdP/view): I created an account in bryophyteportal.org, browsed a few records and quickly found one example (FLAS Bryophytes [30942](https://bryophyteportal.org/portal/collections/individual/index.php?occid=6083273)) that contains habitat information written in German, but recorded in the "associated collectors" field (_Fels, Erde, Mauer, Baum, Strauch, Holz, See, Bach_).
  Somebody understood those were people names because they begin with uppercase (but in German language, all names are written like that). If you translate these words they are obviously related to habitat/substrate: _Rock, soil, wall, tree, bush, wood, lake, stream_ (maybe my translation is not accurate, but you get the idea).  
  My guess is that these labels contained a whole set of common values for those fields, but in these case none of them was checked (i.e. underlined as in [this other record](https://www.gbif.org/occurrence/4076828476)).
  So I have a **new feature request for the crowdsourcing interface**: it does not contain **a field where collaborators can comment**: i.e. my motivations to change something is info provided by me, which is not in the label but might be useful for curators reviewing my edits. IMHO it would be good to have a field where I can say: "_I removed info from 'associated collectors' field; I know it is in German and its translation corresponds to possible habitat/substrate values, but none of them was checked_").  
3. This **crowdsourcing interface** for logged users, is something designed specifically to [globaltcn.utk.edu](https://globaltcn.utk.edu)  portal?  I checked for example [openherbarium.org](https://openherbarium.org) (where I see worlwide institutions) but in that case I don't see a crowdsourcing interface.  
   Is there any basic installation information one can follow to get a running webserver where a particular institution records can be crowsourced like in the case of globaltcn.utk.edu portal? 

Thanks
@abubelinha

**Comment by themerekat** [comment]
Thanks for your questions @abubelinha !

1. On the Symbiota portals page, "Global" means that any collection from anywhere can join this portal. While some digitization projects have been funded for only a subset of collections (e.g., some United States collections, like for the GLOBAL TCN), that does not necessarily mean that the Symbiota portal is exclusively for those collections.

2. Thanks for your thoughts about crowdsourcing metadata! Edits generally only capture who and when, not necessarily why. I do wonder how often a curator would reference the "why" data, time-limited as they are, but it could be helpful for edits that aren't exactly straightforward.

3. Crowdsourcing is possible on all Symbiota portals, but only some portals are actively using it. Do you have data you'd like to get crowdsourced? If so, you could definitely join a portal and get your data in the crowdsourcing queue! More info here: https://biokic.github.io/symbiota-docs/coll_manager/crowdsource/
